CONTRIBUTING.md

```diff
@@ -197,6 +197,10 @@ Furthermore, the pixel shortage is over. We want to see:
 - `grid` instead of `g`
 - `my_function_that_does_things` instead of `mftdt`
 
+### Tips for compilation
+
+By default, libtool builds everything twice (make V=1 shows it), one for the static and oen for the dynamic library, cf. https://stackoverflow.com/questions/572760/libtool-slowness-double-building. If you want to avoid this double compiling time you can run ./configure --disable-static which disables building the static library.
```

Comment:
Sorry for being confusing earlier, `make V=1` is indeed the default. I just made it explicit because had used `make V=0` in #160 . Also "oen" → "one".